src/Contract Test Generator.postman_collection.json

```diff
@@ -1271,7 +1285,7 @@
 									"if(expectedResponse){\r",
 									"    pm.test(`${schemaTest.name} - Has expected response body schema`, function(){\r",
 									"        const Ajv = require('ajv');\r",
-									"        const ajv = new Ajv({allErrors: true,format: false});\r",
+									"        const ajv = new Ajv({allErrors: true,format: false,nullable: true});\r",
```

Comment:
now we actively look at nullable: true.

This way, if the openapi spec says that a field is nullable AND the field is null. We won't receive an error.

I was surprised to see that this is not default behavior of ajv

Reply:
I guess no-one tested that the ajv logic works ... 

```js
const valid = ajv.validate({$ref: `OAS${expectedResponse.$ref}`}, jsonData);
const errors = ajv.errorsText(valid.errors);
```

is wrong according to ajv website. its 

```js
const valid = ajv.validate({$ref: `OAS${expectedResponse.$ref}`}, jsonData);
const errors = ajv.errorsText(ajv.errors);
```

Reply:
I added a fix for schema validation in https://github.com/allenheltondev/postman-contract-test-generator/pull/20 but I suspect other parts using ajv is also wrong.

Reply:
I suppose why no-one didnt notice is because true.errors or false.errors returns undefined. And thus errors will never be  anything else than "No errors"